spec/cloudflare/custom_hostname/ssl_attribute/settings_spec.rb

```diff
@@ -1,54 +1,54 @@
 RSpec.describe Cloudflare::CustomHostname::SSLAttribute::Settings do
 
-  subject { described_class.new({}) }
```

Comment:
This file only has spaces -> tabs changes

Reply:
Thanks for being consistent.